server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -65,15 +70,18 @@
 public class CacheManagerResource implements ResourceHandler {
 
    private final EmbeddedCacheManager cacheManager;
+   private InvocationHelper invocationHelper;
    private final InternalCacheRegistry internalCacheRegistry;
    private final JsonWriter jsonWriter = new JsonWriter();
    private final ObjectMapper objectMapper;
    private final ParserRegistry parserRegistry = new ParserRegistry();
    private final String cacheManagerName;
+   private final static Logger log = LogFactory.getLogger(CacheManagerResource.class.toString());
```

Comment:
never used.

Reply:
yeah, will should add some trace logs TBH